functions/poll.js

```diff
@@ -419,6 +456,14 @@ module.exports = {
 
     return await pollTable(REIMBURSEMENTS_TABLE, STATUS_TO_CALLBACKS);
   }),
+  volunteers: functions.pubsub.schedule('every 2 minutes').onRun(async () => {
```

Comment:
set to two minute schedule as slack API has a [50ish ratelimit](https://api.slack.com/docs/rate-limits) on [`users.lookupByEmail`](https://api.slack.com/methods/users.lookupByEmail)